views/templates/block/activityLog.twig

```diff
@@ -19,6 +19,36 @@
             <div id="errorDuringUpgrade" class="col-xs-12" style="display:none;">
                 <h4>{{ 'Errors'|trans({}, 'Modules.Autoupgrade.Admin') }}</h4>
                 <div id="infoError" class="processing"></div>
+                <button id="reportErrorButton" class="btn btn-warning" style="margin-top: 10px;" data-toggle="modal" data-target="#errorModal">
+                    {{ 'Report errors'|trans({}, 'Modules.Autoupgrade.Admin') }}
+                </button>
+            </div>
+
+            <div id="errorModal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+                <div class="modal-dialog modal-dialog-centered" role="document">
+                    <div class="modal-content">
+                        <div class="modal-header" style="display: flex; justify-content: space-between; align-items: center;">
+                            <h3 class="modal-title">
+                                {{ 'Report errors'|trans({}, 'Modules.Autoupgrade.Admin') }}
+                            </h3>
+                            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                                <span aria-hidden="true">&times;</span>
+                            </button>
+                        </div>
+                        <div class="modal-body">
+                            <input id="userEmail" type="text" class="form-control" placeholder="Email" style="display: block; margin-bottom: 10px;">
+                            <textarea id="errorDescription" rows="3" placeholder="{{ 'Additional details you may think relevant'|trans({}, 'Modules.Autoupgrade.Admin') }}"></textarea>
+                        </div>
+                        <div class="modal-footer">
+                            <button type="button" class="btn btn-secondary" data-dismiss="modal" style="text-transform: uppercase;">
```

Comment:
```suggestion
                            <button type="button" class="btn btn-secondary text-uppercase" data-dismiss="modal">
```

Reply:
Sadly, not work in this version of bootstrap